### PR TITLE
Rename project

### PR DESCRIPTION
**Pull request by colinodell**
See #25

### CODE REVIEW

**Comment by colinodell** [comment]
@hason Would you mind reviewing this? If everything looks good, please merge it in, and then I'll create the new package on Packagist and grant you access to that :)

**Comment by colinodell** [comment]
I don't know why Scrutinizer thinks the tests failed: https://scrutinizer-ci.com/g/thephpleague/commonmark-ext-table/inspections/367798a7-e416-48ef-9f8d-56edf419a3e9 But you should have access to that if you'd like to help investigate further.

**Comment by hason** [comment]
@colinodell Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,71 +1,118 @@
...
+[0.6.1]: https://github.com/thephpleague/commonmark-ext-table/compare/0.6.0...0.6.1
+[0.6.0]: https://github.com/thephpleague/commonmark-ext-table/compare/0.5.0...0.6.0
+[0.5.0]: https://github.com/thephpleague/commonmark-ext-table/compare/0.4.3...0.5.0
+[0.4.3]: https://github.com/thephpleague/commonmark-ext-table/compare/0.4.2...0.4.3
+[0.4.2]: https://github.com/thephpleague/commonmark-ext-table/compare/0.4.1...0.4.2
+[0.4.1]: https://github.com/thephpleague/commonmark-ext-table/compare/0.4.0...0.4.1
+[0.4.0]: https://github.com/thephpleague/commonmark-ext-table/compare/0.3.0...0.4.0
+[0.3.0]: https://github.com/thephpleague/commonmark-ext-table/compare/0.2.0...0.3.0
+[0.2.0]: https://github.com/thephpleague/commonmark-ext-table/compare/0.1.0...0.2.0
+[0.1.0]: https://github.com/thephpleague/commonmark-ext-table/commits/0.2.0
```
Comment (glensc) [inline CHANGELOG.md:118]: should be `commits/0.1.0`

Comment (colinodell) [reply CHANGELOG.md:118]: Fixed via 8b36bf3. Thanks!

**Pull request merged**